tests/unit/local/lambdafn/test_runtime.py

```diff
@@ -168,7 +165,7 @@ def test_must_create_container_first_if_passed_container_is_none(self):
         create_mock.return_value = container
 
         self.runtime.run(None, self.func_config, debug_context=debug_options, event=event)
-        create_mock.assert_called_with(self.func_config, debug_options, event)
+        create_mock.assert_called_with(self.func_config, debug_options)
```

Comment:
Same here, remove the `event` var created on line 156

Reply:
resolved